README.md

```diff
@@ -33,13 +33,6 @@ Check out our [contributing document](https://github.com/libp2p/community/blob/m
 
 <!-- Small note: If editing the README, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification. -->
 
-## Maintainers
```

Comment:
i would leave this and point to the CODEOWNERS, for folks unfamiliar with that mechanism